irohad/ametsuchi/impl/storage_impl.hpp

```diff
@@ -109,6 +109,8 @@ namespace iroha {
                       converter,
                   size_t pool_size);
 
+      void prepareStatements();
```

Comment:
Why it cannot be static (passing connection pool & its size as args)
If the pervious point is positive, than it can removed from interface and have internal linkage